ops/ccs-ops-misc/synthetic-data/scripts/beneficiary-interim-filesplit/split-bene-interim.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+import warnings
+import csv
+import sys
+
+warnings.filterwarnings("ignore")
+
+def split_file(filename):
+    '''Splits beneficiary-interim RIF file to n number of RIF files 
+     with only one beneficiary record per bene_id in each file.
+    '''
```

Comment:
This description is confusing to me and I don't think it has quite enough detail to convey the purpose of this tool and exactly what it does. Is it true that we need this utility to work around a problem where we had the same bene_id appearing multiple times in the same RIF file?

Reply:
Yes, synthea puts all the updates between the first insert and last update into one file, beneficiary_interim. Having multiple updates for the same bene in the same file causes race conditions due to the way the pipeline batches and processes chunks of the update file, possibly at/near the same time. The function of this script is to split that interim file up so that each year of bene updates is contained to one file (i.e. only one unique bene id per file), to ensure the same bene id is not updated at the same time.